includes/REST_API/Stories_Base_Controller.php

```diff
@@ -155,7 +154,7 @@ public function prepare_item_for_response( $post, $request ) {
 	 */
 	public function create_item( $request ) {
 		$kses = new KSES();
-		$kses->init();
+		$kses->add_filters();
```

Comment:
These could keep using `init()`, no?

Reply:
We don't need to capability check here, the rest api does that for us. Also `add_filters` is what `init` used be. This logic is unchanged from before, just different method names.

Reply:
Hmm actually, since `KSES` is now running on `init`, we don't need KSES here anymore, do we? We could just remove this code.